### PR TITLE
Correct targets in IISIntegration.sln and remove TestSites workaround

### PR DESCRIPTION
**Pull request by dougbu**
- not clear either change will clean up CI failures on Win7 and Win2008 but can't hurt
  - don't see other likely suspects
- dotnet/sdk#396 has been fixed; web sites target x86 automatically

nit: sort dependencies

### CODE REVIEW

**Comment by dougbu** [comment]
Small follow-up to your build changes @natemcmaster 
- functional tests currently all fail with `BadImageFormatExceptions` on Win7 and Win Srv 2008R2

**Comment by Tratcher** [comment]
On Win8+ we test the bin deployed ANCM. On Win7 we test the installed ANCM.

**Comment by dougbu** [comment]
@Tratcher will this change help the CI failures? Worthwhile cleanup either way. But, we need to get past this point in the test runs.

**Comment by Tratcher** [comment]
No idea

File: IISIntegration.sln [hunk 1]
```diff
@@ -36,64 +36,64 @@ Global
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{E4E2BDC4-A9C6-4AE9-B429-032EC83EDE64}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{E4E2BDC4-A9C6-4AE9-B429-032EC83EDE64}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{E4E2BDC4-A9C6-4AE9-B429-032EC83EDE64}.Debug|x64.ActiveCfg = Debug|x64
-		{E4E2BDC4-A9C6-4AE9-B429-032EC83EDE64}.Debug|x64.Build.0 = Debug|x64
-		{E4E2BDC4-A9C6-4AE9-B429-032EC83EDE64}.Debug|x86.ActiveCfg = Debug|x86
-		{E4E2BDC4-A9C6-4AE9-B429-032EC83EDE64}.Debug|x86.Build.0 = Debug|x86
+		{E4E2BDC4-A9C6-4AE9-B429-032EC83EDE64}.Debug|x64.ActiveCfg = Debug|Any CPU
```
Comment (natemcmaster) [inline IISIntegration.sln]: Can you undo this change? Mixing x64 and AnyCPU configurations in the solution file may lead to build/test errors.

File: test/TestSites/TestSites.csproj [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
 
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp1.1</TargetFrameworks>
-    <RuntimeIdentifiers>win7-x64;win7-x86;osx.10.10-x64;osx.10.11-x64;osx.10.12-x64;ubuntu.14.04-x64;ubuntu.15.04-x64</RuntimeIdentifiers>
-    <!-- TODO remove when https://github.com/dotnet/sdk/issues/396 is resolved -->
-    <RuntimeIdentifier Condition=" '$(TargetFramework)' != 'netcoreapp1.1' ">win7-x64</RuntimeIdentifier>
     <OutputType>Exe</OutputType>
```
Comment (natemcmaster) [inline test/TestSites/TestSites.csproj]: Nit: you can also remove OutputType. It is the default for Microsoft.NET.Sdk.Web projects.

File: IISIntegration.sln (hunk 1 above)
Comment (dougbu) [inline IISIntegration.sln]: VS did this. Also this change /removes/ x64 by mapping it to AnyCPU.

File: test/TestSites/TestSites.csproj (hunk 2 above)
Comment (dougbu) [inline test/TestSites/TestSites.csproj]: thx

File: IISIntegration.sln (hunk 1 above)
Comment (natemcmaster) [inline IISIntegration.sln]: Yeah, the new project system in VS is super buggy when it comes to solution configurations. They're working on a fixing it in the next update. https://github.com/dotnet/roslyn-project-system/issues/694

Comment (dougbu) [inline IISIntegration.sln]: Talked offline and I'm removing the x64 / x86 configurations from the solution. Testing now.

**Comment by dougbu** [comment]
:up::date:

**Comment by dougbu** [comment]
e367579

**Pull request closed without merging**